layouts/_default/single.html

```diff
@@ -28,7 +28,11 @@ <h1 class="post-title">
   {{ end }}
 
   <div class="post-content">
-    {{ .Content }}
+  {{- with .Content -}}
```

Comment:
Nitpicking but could you fix the indention here? :D

Reply:
Of course. Sorry, my bad.